src/NLog/Targets/FileTarget.cs

```diff
@@ -922,13 +917,13 @@ protected override void InitializeTarget()
         {
             base.InitializeTarget();
 
-            _appenderFactory = GetFileAppenderFactory();
+            var appenderFactory = GetFileAppenderFactory();
```

Comment:
@snakefoot could you please double check this? Thx

Reply:
Looks sane

Reply:
Thanks!